src/common/services/api/cart.service.js

```diff
@@ -89,6 +89,7 @@ class Cart {
       return {
         uri: item.self.uri,
         code: item.itemCode.code,
+        orgId: item.itemDefinition.details[1].value,
```

Comment:
This seems a little fragile. Maybe an `orgId` variable should be set similar to how `designationType` is being set?

Reply:
Ah yeah, I like that approach better. I'll go ahead and update it.